Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version: 5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "NetworkService",
+    platforms: [.iOS(.v15)],
+    products: [
+        .library(
+            name: "NetworkService",
+            targets: ["NetworkService"]),
+    ],
+    dependencies: [],
+    targets: [
+        .target(
+            name: "NetworkService",
+            dependencies: []),
```

Comment:
Please follow the consistent code style:

```suggestion
        .target(
            name: "NetworkService",
            dependencies: []
        ),
```

Reply:
Please review the rest of code.

Reply:
ok